src/Discord.Net.WebSocket/Entities/Channels/SocketVoiceChannel.cs

```diff
@@ -42,9 +42,17 @@ public Task ModifyAsync(Action<VoiceChannelProperties> func, RequestOptions opti
 
         public async Task<IAudioClient> ConnectAsync(Action<IAudioClient> configAction = null)
         {
-            return await Guild.ConnectAudioAsync(Id, false, false, configAction).ConfigureAwait(false);
+            return await Guild.ConnectAudioAsync(Id, false, false, configAction, false).ConfigureAwait(false);
         }
 
+        public async Task<IAudioClient> ConnectAsync(Action<IAudioClient> configAction = null, bool external = false)
+        {
+            return await Guild.ConnectAudioAsync(Id, false, false, configAction, external).ConfigureAwait(false);
```

Comment:
Potentially instead of passing null for deaf/mute here we could add extra params?